src/tools/spin/spinquic.cpp

```diff
@@ -1325,6 +1325,9 @@ CXPLAT_THREAD_CALLBACK(RunThread, Context)
         }
 
         if (0 == GetRandom(4)) {
+            if (!MsQuicLib.LazyInitComplete) {
+                break;
+            }
```

Comment:
This needs to be put in the `library.c` code, not in spinquic.

Reply:
Thanks, will do that